torchvision/ops/boxes.py

```diff
@@ -112,8 +113,16 @@ def clip_boxes_to_image(boxes, size):
     boxes_x = boxes[..., 0::2]
     boxes_y = boxes[..., 1::2]
     height, width = size
-    boxes_x = boxes_x.clamp(min=0, max=width)
-    boxes_y = boxes_y.clamp(min=0, max=height)
+
+    if torchvision._is_tracing():
```

Comment:
Could we add an individual test for this in ONNX, so that we can be sure that the function doesn't break in ONNX in the future?